app/src/entry-point.lisp

```diff
@@ -327,7 +349,21 @@ Copyright (c) 2016-2019 Rigetti Computing.~2%")
                           shared
                           simulation-method
                           #-forest-sdk debug
-                          #+forest-sdk skip-version-check)
+                          #+forest-sdk skip-version-check
+                          quiet
+                          log-level)
+
+  (setf *logger* (make-instance 'cl-syslog:rfc5424-logger
+                                :app-name "qvm"
+                                :facility ':local0
+                                :maximum-priority (log-level-string-to-symbol log-level)
+                                :log-writer
+                                #+windows
+                                (cl-syslog:stream-log-writer)
```

Comment:
Should this also be conditionalized on `quiet`?

Reply:
I don't think so. quiet is for non-logging output like the banner

Reply:
o, "non-logging". ok!